docs/whatsnew/1.0.rst

```diff
@@ -146,6 +149,55 @@ fast version, in which case the pure-Python version will automatically be used.
 
 For full details including extensive performance testing, see :ref:`fast_ascii_io`.
 
+Enhanced CSV format
+^^^^^^^^^^^^^^^^^^^
+
+One of the problems when storing a table in an ASCII format is preserving table
+meta-data such as comments, keywords and column data types, units, and
+descriptions.  Using the newly defined `Enhanced Character Separated Values
+format <https://github.com/astropy/astropy-APEs/blob/master/APE6.rst>`_ it is
+now possible to write a table to an ASCII-format file and read it back with no
+loss of information.  The ECSV format has been designed to be both
+human-readable and compatible with most simple CSV readers.
+
+In the example below we show writing a table that has ``float32`` and ``bool``
+types.  This illustrates the simple look of the format which has a few header
+lines (starting with ``#``) in `YAML <http://www.yaml.org/>`_ format and then
```

Comment:
Do we want to mention that the new feature requires PyYAML?  I know it's mentioned in `install.rst`, but maybe here as well to help people who just want to start playing with this?